Tests/Build-Bicep.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+try {
+    $ScriptDirectory = Split-Path -Path $PSCommandPath -Parent
+    Import-Module -FullyQualifiedName "$ScriptDirectory\..\Source\Bicep.psd1"
+}
+catch {
+    Throw "Unable to import Bicep module. $_"
+}
+
+InModuleScope Bicep {
+    BeforeAll {
+        $ScriptDirectory = Split-Path -Path $PSCommandPath -Parent
+        Copy-Item "$ScriptDirectory\supportFiles\*.bicep" -Destination TestDrive:\
+    }
+    Describe 'Build-Bicep' {
+        
+        Context 'When it works' {            
+            It 'Build a bicep file AsString' {
+                Build-Bicep -Path 'TestDrive:\workingBicep.bicep' -AsString | Should -Not -BeNullOrEmpty
+            }
+            It 'Build a bicep file AsHashtable' {
+                Build-Bicep -Path 'TestDrive:\workingBicep.bicep' -AsHashtable | Should -Not -BeNullOrEmpty
+            }
+            It 'Build a bicep file' {
+                $file='TestDrive:\workingBicep.bicep'
+                Build-Bicep -Path $file
+                $templateFile = $file -replace '\.bicep', '.json'
+                Get-Content -Path $templateFile -Raw | Should -Not -BeNullOrEmpty
```

Comment:
Can we also do ConvertFrom-Json to test that the output is valid json?
It would be awesome to also validate it as a valid ARM template, but that might be overkill